<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1001</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Horizontal">
     <attribute name="title">
      <string>Sismógrafo horizontal (Lehman)</string>
     </attribute>
     <widget class="PlotWidget" name="graphViewLehmanSignal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>981</width>
        <height>291</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorViewCenter</enum>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>981</width>
        <height>291</height>
       </rect>
      </property>
      <widget class="PlotWidget" name="graphViewLehmanFourier">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>981</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="valueLehmanFFTB">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="valueLehmanSignalX">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="valueLehmanFFTA">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>330</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="valueLehmanSignalY">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Vertical">
     <attribute name="title">
      <string>Sismógrafo vertical (LaCoste)</string>
     </attribute>
     <widget class="PlotWidget" name="graphViewLaCosteSignal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>981</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="PlotWidget" name="graphViewLaCosteFourier">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>981</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="valueLaCosteFFTA">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>330</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="valueLaCosteSignalX">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="valueLaCosteSignalY">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="valueLaCosteFFTB">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>360</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
